wpengine/faustjs
# Chore: Update Block Support

## Tasks


## Description

- Updates dependencies of Block Support example.
- Fixes issue with pages not rendering inner blocks.

## Related Issue(s):



## Testing

- Make sure you checkout this PR in Content Blocks: https://github.com/wpengine/wp-graphql-content-blocks/pull/182
- Install dependencies
```bash
npm i
```
- Delete to node_modules inside the block_support folder to avoid any issues with Multiple React Versions:

```bash
rm -rf examples/next/block-support/node_modules 
```

- Make sure you have setup a home page as a single page/post in WordPress Reading Settings:
<img width="885" alt="Screenshot 2024-01-25 at 16 32 54" src="https://github.com/wpengine/faustjs/assets/328805/ac7bd214-d8a0-4d60-974b-81c3a41da4a8">

- Update the blockset command:

```
npm run blockset -w examples/next/block-support
```

- Run the project and you should be able to see the front-page without issues.

```bash
npm run dev -w examples/next/block-support
```

## Screenshots



## Documentation Changes



## Dependant PRs
